lightly_studio/tests/core/test_add_samples.py

```diff
@@ -341,6 +325,35 @@ def test_tag_samples_by_directory_tag_depth_1(
     assert sample_filename_to_tags["root_img.png"] == set()
 
 
+def test_add_samples_from_path_calls_tag_samples_by_directory(
+    db_session: Session,
+    tmp_path: Path,
+    mocker: Mocker,
+) -> None:
+    """Tests that Dataset.add_samples_from_path correctly calls the helper.
+
+    The add_samples.tag_samples_by_directory helper.
+    """
+    mocker.patch("lightly_studio.core.dataset._generate_embeddings")
+    spy_tagger = mocker.spy(add_samples, "tag_samples_by_directory")
+
+    dataset_table = helpers_resolvers.create_dataset(db_session, "test_dataset")
+    dataset = Dataset(dataset=dataset_table)
+    dataset.session = db_session
+
+    _create_sample_images([tmp_path / "image1.jpg"])
+
+    dataset.add_samples_from_path(path=str(tmp_path), tag_depth=0)
```

Comment:
I did not test it, but is setting `embed=False` sufficient, so that `mocker.patch("lightly_studio.core.dataset._generate_embeddings")` is not needed?

```suggestion
    spy_tagger = mocker.spy(add_samples, "tag_samples_by_directory")

    dataset_table = helpers_resolvers.create_dataset(db_session, "test_dataset")
    dataset = Dataset(dataset=dataset_table)
    dataset.session = db_session

    _create_sample_images([tmp_path / "image1.jpg"])

    dataset.add_samples_from_path(path=str(tmp_path), tag_depth=0, embed=False)
```

Feel free to resolve this comment yourself if it does not work.